sdk/appconfiguration/Azure.ApplicationModel.Configuration/src/Azure.Configuration.csproj

```diff
@@ -10,6 +10,8 @@
       ]]>
     </PackageReleaseNotes>
 
+      <LangVersion>8.0</LangVersion>
```

Comment:
This will now generate code that cannot be used from older versions of the compiler. Let's wait till we have the nuget packkage before merging this PR.

Reply:
Also note that we automatically default to latest in the common props so hopefully this shouldn't be needed.

Reply:
New features require `LangVersion=preview`